supervisor/erratum_handler.py

```diff
@@ -1,5 +1,5 @@
 import logging
-from datetime import datetime, timedelta, timezone
```

Comment:
I don't like having this and the reindentation in the same commit as the  match changes - can you split them out? @panyamkeerthana - we should figure out why you aren't getting the same indentation as @Jazzcort  (and me) - I have vscode configured to use the Black formatter with --line-length=88 (which is I think the default.)